QUEST_LV_0100.tsv

```diff
@@ -641,23 +641,23 @@ QUEST_LV_0100_20150317_000640	Coben
 QUEST_LV_0100_20150317_000641	$I am looking for my brother.{nl}If you can help me, I may give you a reward.
 QUEST_LV_0100_20150317_000642	$He was a really fantastic person.{nl}I can't imagine myself being like him.
 QUEST_LV_0100_20150317_000643	$Something is between the pages of this diary.{nl}A blueprint.. and oh, my brother had recorded a few things here.
-QUEST_LV_0100_20150317_000644	$I came here to look for my brother, but I am not as brave as he was.{nl}I want you to take a look at his gravestone's writing on my behalf.
-QUEST_LV_0100_20150317_000645	$Yes. My brother was a thief.{nl}My brother used to come to our shop once in a few years and tell us stories about his adventures.
-QUEST_LV_0100_20150317_000646	$So he challenged the Archmage Tower?{nl}That's my brother! He could certainly do that.{nl}
+QUEST_LV_0100_20150317_000644	$I came here to look for my brother, but I am not as brave as he is.{nl}I want you to take a look at his monuments' writing on my behalf.
+QUEST_LV_0100_20150317_000645	$Yes. My brother is a thief.{nl}My brother used to come to our shop once in a few years and tell us stories about his adventures.
+QUEST_LV_0100_20150317_000646	$So he challenged the great Mage Tower?{nl}That's my brother! He could certainly do that.{nl}
```

Comment:
QUEST_LV_0100_20150317_000646	is a dialog involving the brother that's trying to invade the Mage Tower (he died to Agailla Flurry). I'm pretty sure he isn't invading the instanced dungeon, which has no story driven for it.

The very next line that comes after this line is:
QUEST_LV_0100_20150317_000647	$The Mage Tower was a fortress that demons could not even try to approach.

So the speaker (the quest giver) is actually referring to the Mage Tower, not the Archmage Tower.